core/templates/domain/improvements/exploration-task.model.spec.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Unit tests for the ExplorationTaskModel.
  */
 
-import { ExplorationTaskModel } from
+import { ExplorationTaskBackendDict, ExplorationTaskModel } from
   'domain/improvements/exploration-task.model';
```

Comment:
```suggestion
import { ExplorationTaskBackendDict, ExplorationTaskModel } from 'domain/improvements/exploration-task.model';
```
Change imports to a single line. Do the same for the rest of the imports in this file.

Reply:
done